scripts/Facilities.js

```diff
@@ -0,0 +1,19 @@
+import {getFacilities, getMinerals} from "./database.js"
+
+const facilities = getFacilities()
+const minerals = getMinerals
+
+export const facilityButtonHTMLDisplayer = () => {
+    let html = "<section class='facilities--flex'>"
+    const facilityDivsForHTML = facilities.map(facility => {
```

Comment:
I wouldn't add anything about HTML when defining a variable, it can be confusing when its right near the "let html =" and "return html"